### DISCUSSION TITLE
Πρότυπο (template) για αίτημα ενσωμάτωσης της τελικής αναφοράς

### DISCUSSION THREAD

**Discussion by AlexandrosV20** [courses-ionio/hci#1677] [Announcements]
Θέλω να ρωτήσω το κείμενο με τα τικ, που πρέπει να το προσθέσουμε στο pull request, που το βρίσκουμε ;
![image](https://user-images.githubusercontent.com/17967496/149187726-01a71cca-4e94-4e95-9452-94fe8ea0245d.png)

**Comment by Sola1r3** [comment]
Έχω την εντύπωση πως εμφανίζεται αυτόματα όταν κάνεις pull request, δεν πρέπει να το πάρεις από κάπου.

**Reply by AlexandrosV20** [reply]
ισχυει οντως ετσι ειναι... σε ευχαριστω πολυ για την βοηθεια
